packages/core/src/lib/translate/translate.ts

```diff
@@ -7,6 +7,8 @@ import vi from "../locale/vi.json";
 import hi from "../locale/hi.json";
 import ar from "../locale/ar.json";
 
+const allowOnlyLanguage = null; // leave null or change into language code "en", "es"...
```

Comment:
How are developers going to change the value of this `allowOnlyLanguage` variable on their dApp?

Reply:
@kujtimprenkuSQA, thanks for your comment. 
@HarisSQA   `allowOnlyLanguage`  should be put in function that is exported, so language as an argument could be passed to `translate.js`. from dApp.

Reply:
Done. Is now okay?